api/mime.go

```diff
@@ -63,6 +66,8 @@ func FormatToMediaType(format string) string {
 		return MediaTypeCSV
 	case "json":
 		return MediaTypeJSON
+	case "line":
+		return MediaTypeLine
```

Comment:
From what I can tell by looking at the code, the mapping in this direction would not actually be used anywhere. But I assume we prefer the symmetry.

Reply:
🤷🏻 Doesn't hurt.